code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -110,6 +110,13 @@
 	ammo_type = /obj/item/ammo_casing/shotgun/tranquilizer
 	materials = list(MAT_METAL=1750)
 
+/obj/item/ammo_box/tranqdarts

```

Comment:
I'm willing to bet this looks like a box, but behaves like a speedloader. This is inconsistent with the way things work currently on Paradise.